### DISCUSSION TITLE
Uploading a file

### DISCUSSION THREAD

**Discussion by DarkTerrorWolf** [spatie/dropbox-api#83] [Q&A]
I have been playing in the api for the last 2 weeks and i am unable to figure out how to get the api to upload a file. even the test upload throws a bad request error. How would i be able to upload a pdf style file from a remote server to the dropbox?

**Comment by andrejIka** [comment]
Something like this:

```
$authorizationToken = 'token';
$client = new Spatie\Dropbox\Client($authorizationToken);
$path = './db.sql';
$client->upload('/db.sql', file_get_contents($path), 'overwrite', false);
```
